doc/changes/changes_2.0.1.md

```diff
@@ -0,0 +1,20 @@
+# bucketfs-java 2.0.0, released 2021-04-29
+
+Code name: Improved repeated upload reliability
+
+## Summary
+
+In certain situations especially on 6.2.x there was a race condition that made detection of successful reupload to the same path in a bucket unreliable.
+This has now been fixed. This scenario is realistically only happening in integration tests, but there it is likely.
+
+Replaced handwritten dependency list with auto-generated list and linked it in the [README](../../README.md).
```

Comment:
The link won't work if you copy this to the release announcement.

Reply:
Removed link. It is not important anyway.